ArduCopter/Copter.h

```diff
@@ -62,11 +62,11 @@
 #include <AC_InputManager/AC_InputManager_Heli.h>   // Heli specific pilot input handling library
 #include <AP_Arming/AP_Arming.h>
 #include <AP_SmartRTL/AP_SmartRTL.h>
```

Comment:
Why not add a comment to these as well as `AP_Parachute.h` and `AP_Proximity.h` below and finish it up :) ?
```suggestion
#include <AP_Arming/AP_Arming.h>
#include <AP_SmartRTL/AP_SmartRTL.h>
```